server/src/input/pdfminer/pdfminer.ts

```diff
@@ -224,17 +212,17 @@ function getValidCharacter(character: string): string {
 
 function interpretImages(
   fig: PdfminerFigure,
-  imagsLocation: string,
+  imagesLocation: string,
   pageHeight: number,
   scalingFactor: number = 1,
 ): Image[] {
-  return fig.image.map(
-    (img: PdfminerImage) =>
-      new Image(
-        getBoundingBox(fig._attr.bbox, ',', pageHeight, scalingFactor),
-        path.join(imagsLocation, img._attr.src),
-      ),
-  );
+    return fig.image.map(
+      (img: PdfminerImage) =>
+        new Image(
+          getBoundingBox(fig._attr.bbox, ',', pageHeight, scalingFactor),
+          img._attr.src !== undefined ? path.join(imagesLocation, img._attr.src) : "",
```

Comment:
If pdfminer does not extract images anymore, does the `img._attr.src` make sense? Will it be !== undefined in any case?

Reply:
Done in 93fc4cc